linter/linter.go

```diff
@@ -205,9 +212,9 @@ func (linter *Linter) fmtList(header string, list []error) string {
 	return str
 }
 
-func (linter *Linter) printCheckResult(header string, result *spec.CheckResult) {
+func (linter *Linter) printCheckResult(header string, result *spec.CheckResult) bool {
```

Comment:
I'd name the return value to make it clear what it's indicating — `printed` or `didPrint` or something like that.